### PR TITLE
Pods

### PR DESCRIPTION
**Pull request by cnlangzi**
## Changes
- show pods list
- show pod events for checking if it crashed

### CODE REVIEW

File: deploy-0chain/action.yml [hunk 1]
```diff
@@ -379,20 +379,7 @@ runs:
...
-            echo '#!/bin/bash
-            RED="\033[0;31m"
-            NC="\033[0m"
-            echo
-            echo -e "${RED}======================================================${NC}"
-            echo -e "${RED}RETRYING DEPLOYMENT ($(echo $1 | awk '"'"'{ print toupper($0) }'"'"') ATTEMPT):${NC}"
-            echo -e "${RED}======================================================${NC}"
-            echo
-            ./teardown.sh && ./deploy.sh && cd $GITHUB_WORKSPACE && ./stake_blobbers.sh $1;
-            ' > ./retry.sh;
```
Comment (stewartie4) [inline deploy-0chain/action.yml:393]: This retry logic is important for stable CI 
We shouldn't be removing it

**Review by stewartie4 — CHANGES_REQUESTED** [review]

File: deploy-0chain/action.yml (hunk 1 above)
Comment (cnlangzi) [reply deploy-0chain/action.yml:393]: fixed

File: run-system-tests/action.yml [hunk 2]
```diff
@@ -414,44 +414,70 @@ runs:
       if: ${{ failure()}}
       shell: 'script --return --quiet --command "bash {0}"'
       run: |
+        echo
+        echo "======================================================"
+        echo "Show Blobbers:"
+        echo "======================================================"
+
```
Comment (stewartie4) [inline run-system-tests/action.yml:421]: should this be here?

Comment (cnlangzi) [reply run-system-tests/action.yml]: fixed

File: deploy-0chain/action.yml [hunk 3]
```diff
@@ -378,23 +383,23 @@ runs:
...
+          echo
+          ./teardown.sh && ./deploy.sh && cd $GITHUB_WORKSPACE && ./stake_blobbers.sh $1;
+          ' > ./retry.sh;
+          chmod 777 ./retry.sh;
+          timeout 300 ./retry.sh "second" || cd $GITHUB_WORKSPACE && cat build.log && cd .. && timeout 300 ./retry.sh "final" || cd $GITHUB_WORKSPACE && cat build.log && exit 1;
+        else
+          echo "Blobbers staked succefully on try $((ITER+1)). Moving on to tests..."
         fi
 
     - name: "Extract Logging From Pods"
```
Comment (stewartie4) [inline deploy-0chain/action.yml:405]: I added to the if below in my PR
https://github.com/0chain/actions/pull/56/files#diff-f4b0c0a5824ad0fb33cd15ecc1fbc29fa819ad9d948948b7c8f9eec5fbc00459R384
so we only do this if we have deployed a network out as part of the test run (as opposed to for example running against dev)

**Review by stewartie4 — APPROVED** [review]

File: deploy-0chain/action.yml (hunk 3 above)
Comment (cnlangzi) [reply deploy-0chain/action.yml:405]: fixed

**Pull request merged**